### ISSUE TITLE
ut: TestRouterNotFound error

### ISSUE THREAD

**Issue by thinkerou** [gin-gonic/gin#1058]
https://travis-ci.org/gin-gonic/gin/jobs/262229709

```
--- FAIL: TestRouterNotFound (0.00s)
	Error Trace:	routes_test.go:377
	Error:		Not equal: 
			expected: "map[Content-Type:[text/html; charset=utf-8] Location:[/path]]"
			received: "map[Location:[/path]]"
```

**Comment by easonlin404** [comment]
It's kind of weird. I had same issue with: https://travis-ci.org/gin-gonic/gin/jobs/263774413

**Comment by easonlin404** [comment]
fixed by https://github.com/gin-gonic/gin/pull/1063

**Comment by thinkerou** [comment]
closed via #1064
